tests/featureTesting/compare_json.py

```diff
@@ -0,0 +1,149 @@
+import constants as cs
+
+import helper_functions as hf
+
+import pandas as pd
+
+
+class CompareJsons(object):
+    def __init__(self, man_path, auto_path, analysis_type = cs.by_delta_rt_and_mz, n_rt=5,
+                 n_mz = 2,delta_rt = 0.2, delta_mz =0.3,corr_cutoff = 1, p_val_cutoff = 0.7,
+                 comparater_list= None,unique_identifier_list= None):
+        if unique_identifier_list is None:
+            unique_identifier_list = []
+        if comparater_list is None:
+            comparater_list = []
+
+        self.man_path = man_path
+        self.auto_path = auto_path
+        self.analysis_type = analysis_type
+        self.n_rt = n_rt
+        self.n_mz = n_mz
+        self.delta_rt = delta_rt
+        self.delta_mz = delta_mz
+        self.corr_cutoff = corr_cutoff
+        self.p_val_cutoff = p_val_cutoff
+        self.comparater_list = comparater_list
+        self.unique_identifier_list = unique_identifier_list
+
+
+
+    def summary_report(self):
+        correlation_and_wilcox_df = self.correlation_and_wilcox()
+        correlation_plot_url = self.correlation_plot()
+
+
+
```

Comment:
Remove redundant space and try to follow PEP8 as closely as possible. These things can be taken care of.

Reply:
@harshitcodes 👍